Codespaces (#50)
ImperialCollegeLondon/grad_school_software_engineering_course: setup.md

episodes, we will include indications on what are the equivalent tools and approaches in
other programming languages, such that interested readers can check what they should use 
in those cases.

Some useful links to integrate the Fortran and C++ tools that will be mentioned during the
course can be found in:

- **C++**: [C/C++ VSCode extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools), including a [full tutorial](https://code.visualstudio.com/docs/languages/cpp) on how to set VSCode to work with C++., but for Windows it might be easier to just use Visual Studio.
- **Fortran**: [VSCode Modern Fortran extension](https://fortran-lang.github.io/vscode-fortran-support/), which integrates with the linters and formatters and the Fortran Language Server [fortls](https://fortls.fortran-lang.org/).

## Codespaces

This course is designed to teach you several fundamental concepts on software engineering
good practices as well as helping you to setup your computer so you can keep developing your
own code following these principles once the course is over. However, there are occasions

> when, vor a variety of reasons, installing and running the above tools and those described

Reviewer: ```suggestion
when, for a variety of reasons, installing and running the above tools and those described
```

- **Fortran**: [VSCode Modern Fortran extension](https://fortran-lang.github.io/vscode-fortran-support/), which integrates with the linters and formatters and the Fortran Language Server [fortls](https://fortls.fortran-lang.org/).

## Codespaces

> This course is designed to teach you several fundamental concepts on software engineering

Reviewer 2: ```suggestion
This course is designed to teach you several fundamental concepts from software engineering
```

good practices as well as helping you to setup your computer so you can keep developing your
own code following these principles once the course is over. However, there are occasions
when, for a variety of reasons, installing and running the above tools and those described
in the course just does not work as expected.

> Rather than holding your learning or that of your colleagues, if after a few minutes

Reviewer 2: ```suggestion
Rather than holding up your learning or that of your colleagues, if after a few minutes
```

> trying to diagnose and solve the problem things are not working, you can run the exercises

Reviewer 2: ```suggestion
trying to solve the problem things are still not working, you can run the exercises
```
